src/site/stages/build/drupal/menus.js

```diff
@@ -265,10 +265,9 @@ function makeSection(hostUrl, hub, arrayDepth, promo, pages) {
  * @return {Array} headerData - Menu information formatted for the megaMenu React widget.
  */
 function formatHeaderData(buildOptions, contentData) {
-  if (!contentData?.data) {
+  if (!contentData?.data?.menuLinkContentQuery?.entities) {
     // eslint-disable-next-line no-console
-    console.warn('formatHeaderData has no data');
-    return null;
+    throw new Error('formatHeaderData has no data');
```

Comment:
I made this check stronger because I just can't imagine a case where it is acceptable for a build to report success when the MegaMenu is actually failing to build.